deps/build.jl

```diff
@@ -21,6 +21,7 @@ try
         Rhome = get(ENV, "R_HOME", "")
         if Rhome == "*"
             # install with Conda
+            different = "  To use a different R installation, set the \"R_HOME\" environment variable and re-run Pkg.build(\"RCall\")."
             @info "Installing R via Conda.$different"
```

Comment:
since it is only used here, maybe just add it directly to the message to avoid confusion.